packages/legend-application-studio/src/components/editor/edit-panel/uml-editor/StereotypeSelector.tsx

```diff
@@ -86,55 +107,121 @@ export const StereotypeSelector = observer(
     };
     const updateStereotype = (val: StereotypeOption): void =>
       stereotypeReference_setValue(stereotype, val.value);
+
+    // Drag and Drop
+    const handleHover = useCallback(
+      (item: AllStereotypeDragSource, monitor: DropTargetMonitor): void => {
```

Comment:
Let's rename `AllStereotypeDragSource` --> `StereotypeDragSource` and move it to this file